phpstan-baseline.neon

```diff
@@ -2745,6 +2760,11 @@ parameters:
 			count: 1
 			path: app/Http/Controllers/SubmissionController.php
 
+		-
+			message: "#^Method App\\\\Http\\\\Controllers\\\\SubmissionController\\:\\:submitProcess\\(\\) throws checked exception App\\\\Exceptions\\\\CDashXMLValidationException but it's missing from the PHPDoc @throws tag\\.$#"
+			count: 1
+			path: app/Http/Controllers/SubmissionController.php
+
```

Comment:
We should add `@throws` annotations whenever custom CDash exceptions are being thrown.